components/lib/fetch-okhttp/src/main/java/mozilla/components/lib/fetch/okhttp/OkHttpClient.kt

```diff
@@ -37,6 +37,10 @@ class OkHttpClient(
     )
 
     override fun fetch(request: Request): Response {
+        if (request.private) {
+            throw IllegalStateException("Client doesn't support private request")
```

Comment:
nit: This should be `IllegalArgumentException`. `IllegalStateException` is used to indicate that the method can't be invoked.

Reply:
👍🏽